README.md

```diff
@@ -69,6 +69,24 @@ snakeCase("foo-barBaz");
 // foo_bar_baz
 ```
 
+### `flatCase`
+
+Splits string and joins by flatcase convention:
+
+```ts
+flatCase("foo-barBaz");
+// foobarbaz
+```
+
+### `trainCase`
+
+Split string and joins by Train-Case (a.k.a. HTTP-Header-Case) convention:
```

Comment:
Interesting. I have never heared about this convention name before. I found https://stackoverflow.com/questions/17326185/what-are-the-different-kinds-of-cases saying `TRAIN-CASE` is alias to `COBOL-CASE`, it is upper merge by `-`. Do you have any references that it is like this?

Reply:
See below comment 😅

Reply:
I did a quick research and found 

- https://amydegregorio.com/2020/10/25/programming-case-types/ and
- https://medium.com/@code.ceeker/naming-conventions-camel-case-pascal-case-kebab-case-and-more-dc4e515b9652

Cobol-case is bascially screaming kebap-case or uppercase train-case, which is different from train-case.

All the other mentioned cases can be created by upper- or lowercasing one of the existing utilities (screming snake case, etc.). Maybe we can provide an additional optional parameter in the future for type-safe upper and lowercase based on string-ts. This would enable to also create

- SCREAMING_SNAKE_CASE (`capitalize(snakeCase('FooBar'))`)
- UPPERFLATCASE (`capitalize(flatCase('FooBar'))`)
- COBOL-CASE (`capitalize(kebapCase('FooBar'))`)

with nearly no additional effort.
For Pascal_Snake_Case and camel_Snake_Casem I have never seen them in the wild tbh.

Reply:
Fair enough thanks for references we can assume `Train-Case` is like this.

For more utils, I guess it is worth to be added here or later because, yes, with the core functionality of split string in sclue all other (current) utils could be one liner examples too. We expose them for convention.